src/Avalonia.Controls/DateTimePickers/TimePickerPresenter.cs

```diff
@@ -1,43 +1,42 @@
-﻿using Avalonia.Controls.Metadata;
+﻿using System;
+using Avalonia.Controls.Metadata;
 using Avalonia.Controls.Primitives;
-using Avalonia.Controls.Shapes;
 using Avalonia.Input;
 using Avalonia.Interactivity;
-using System;
 
 namespace Avalonia.Controls
 {
     /// <summary>
     /// Defines the presenter used for selecting a time. Intended for use with
     /// <see cref="TimePicker"/> but can be used independently
     /// </summary>
-    [TemplatePart("PART_AcceptButton",     typeof(Button), IsRequired = true)]
-    [TemplatePart("PART_DismissButton",    typeof(Button))]
-    [TemplatePart("PART_HourDownButton",   typeof(RepeatButton))]
-    [TemplatePart("PART_HourSelector",     typeof(DateTimePickerPanel), IsRequired = true)]
-    [TemplatePart("PART_HourUpButton",     typeof(RepeatButton))]
-    [TemplatePart("PART_MinuteDownButton", typeof(RepeatButton))]
-    [TemplatePart("PART_MinuteSelector",   typeof(DateTimePickerPanel), IsRequired = true)]
-    [TemplatePart("PART_MinuteUpButton",   typeof(RepeatButton))]
-    [TemplatePart("PART_SecondDownButton", typeof(RepeatButton))]
-    [TemplatePart("PART_SecondHost",       typeof(Panel), IsRequired = true)]
-    [TemplatePart("PART_SecondSelector",   typeof(DateTimePickerPanel), IsRequired = true)]
-    [TemplatePart("PART_SecondUpButton",   typeof(RepeatButton))]
-    [TemplatePart("PART_PeriodDownButton", typeof(RepeatButton))]
-    [TemplatePart("PART_PeriodHost",       typeof(Panel), IsRequired = true)]
-    [TemplatePart("PART_PeriodSelector",   typeof(DateTimePickerPanel), IsRequired = true)]
-    [TemplatePart("PART_PeriodUpButton",   typeof(RepeatButton))]
-    [TemplatePart("PART_PickerContainer",  typeof(Grid), IsRequired = true)]
-    [TemplatePart("PART_SecondSpacer",     typeof(Rectangle), IsRequired = true)]
-    [TemplatePart("PART_ThirdSpacer",      typeof(Rectangle), IsRequired = true)]
+    [TemplatePart(nameof(TemplateItems.PART_AcceptButton),      typeof(Button), IsRequired = true)]
```

Comment:
I'm not really a big fan of this. While I like `nameof()` in general, these constants are part of the public API, whereas the private fields are not. Renaming some private field should definitely not change the public API. Could you please revert to string constants?

Once that's done, change to the fields so they're not prefixed by `PART_`, matching the standard naming convention.